Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -5,6 +5,137 @@ import XCTest
+
+        //GIVEN
+        let traffic = Traffic(
+            key: "key",
+            segments: .plain("segment"),
+            percentage: 13,
+            allocation: []
+        )
+        let overrideFeature = OverrideFeature(
+            enabled: false,
+            variation: "variation",
+            variables: [:]
+        )
+        let variation = Variation(
+            description: "description",
+            value: "value",
+            weight: 3,
+            variables: []
+        )
+        let variableSchema = VariableSchema(
+            key: "key",
+            type: .object,
+            defaultValue: .object(["": .boolean(false)])
+        )
+
+        let evaluation = Evaluation(
+            featureKey: "feature123",
+            reason: .allocated,
+            bucketValue: 42,
+            ruleKey: "rule456",
+            enabled: true,
+            traffic: traffic,
+            sticky: overrideFeature,
+            initial: overrideFeature,
+            variation: variation,
+            variationValue: "value",
+            variableKey: "color",
+            variableValue: .string(""),
+            variableSchema: variableSchema
+        )
+
+        let mockedTraffic: [String: Any] = [
+            "allocation": [Any](),
+            "key": "key",
+            "percentage": 13,
+            "segments": [
+                "plain": [
+                    "_0": "segment"
+                ]
+            ],
+        ]
+
+        let mockedOverrideFeature: [String: Any] = [
+            "enabled": false,
+            "variables": [String: Any](),
+            "variation": "variation",
+        ]
+
+        let mockedVariation: [String: Any] = [
+            "description": "description",
+            "value": "value",
+            "variables": [Any](),
+            "weight": 3,
+        ]
+
+        let mockedVariableSchema: [String: Any] = [
+            "key": "key",
+            "defaultValue": ["": 0],
+            "type": "object",
+        ]
+
+        let expectedDictionary: [String: Any] = [
+            "featureKey": "feature123",
+            "reason": "allocated",
+            "bucketValue": 42,
+            "ruleKey": "rule456",
+            "enabled": true,
+            "traffic": mockedTraffic,
+            "sticky": mockedOverrideFeature,
+            "initial": mockedOverrideFeature,
+            "variation": mockedVariation,
+            "variationValue": "value",
+            "variableKey": "color",
+            "variableValue": "",
+            "variableSchema": mockedVariableSchema,
+        ]
+
+        let encoder = JSONEncoder()
+        let jsonData = try encoder.encode(evaluation)
+
+        //WHEN
+        guard
+            let decodedDictionary = try JSONSerialization.jsonObject(with: jsonData, options: [])
+                as? [String: Any]
+        else {
+            XCTFail("Failed to decode JSON to a dictionary.")
+            return
+        }
+
```

Comment:
You don't have "THEN" where you can assert vs `expectedDictionary` as now it's not being used